src/components/loadable.cjsx

```diff
@@ -15,6 +15,13 @@ module.exports = React.createClass
     isLoading: React.PropTypes.func.isRequired
     isLoaded: React.PropTypes.func.isRequired
     isFailed: React.PropTypes.func.isRequired
+    renderStatus: React.PropTypes.func.isRequired
+
+  getDefaultProps: ->
+
+    # Enables a renderStatus prop function with a component other than a div
+    renderStatus: (status, message) ->
```

Comment:
Why not have props named `renderLoading`, `renderError`, or `renderBug` (and react can statically check their types to make sure they're a component) and have default props for each?

That way components can just provide a custom status when needed rather than implement `renderStatus` with a switch statement?